plugins/autolabor/autohauler.cpp

```diff
@@ -86,37 +77,29 @@ REQUIRE_GLOBAL(world);
  * autolabor.plug.dll as this plugin is mutually exclusive with it.
  */
 
-// Yep...don't know what it does
 DFHACK_PLUGIN_IS_ENABLED(enable_autohauler);
 
-// This is the configuration saved into the world save file
-static PersistentDataItem config;
-
-// There is a possibility I will add extensive, line-by-line debug capability
-// later
 static bool print_debug = false;
```

Comment:
it doesn't need to be done in this PR, but nowadays debug printing is handled by the central DFHack debug logging. See https://github.com/DFHack/dfhack/blob/develop/plugins/examples/persistent_per_save_example.cpp#L66

Reply:
thanks for reminding me